app/views/user_mailer/invitation.html.erb

```diff
@@ -12,15 +12,19 @@
   </p>
 
   <p>
-    <%= t ".part1" %>
+    <%= t(".html.ao3_intro", otw_link: style_link("Organization for Transformative Works", "http://transformativeworks.org")).html_safe %>
   </p>
 
   <p>
-    <%= t ".part2" %>
+    <%= t ".account_functionality" %>
   </p>
 
   <p>
-    <%= t(".html.part3", invitation_link: style_link(signup_url(:invitation_token => @invitation.token), signup_url(:invitation_token => @invitation.token))).html_safe %>
+    <%= t(".html.invite_link", invitation_link: style_link(signup_url(:invitation_token => @invitation.token), signup_url(:invitation_token => @invitation.token))).html_safe %>
```

Comment:
This isn't quite right. `invitation_link` uses the `style_link` helper to produce the HTML for a link using the argument as text and the second argument as the URL, and in this case it's going to look something like the following:

`<a href="https://archiveofourown.org/signup/keysmash" style="color:#900">https://archiveofourown.org/signup/keysmash</a>`

Which, when combined with the text you've provided for `.html.invite_link`, will produce roughly:

`If you'd like to join us, please <a href='<a href="https://archiveofourown.org/signup/keysmash" style="color:#900"> https://archiveofourown.org/signup/keysmash</a>'> follow this link to sign up</a>.`

What we need here in the view here is something more like:

`<%= t(".html.join", invitation_link: style_link(t(".html.invitation_link_text")), signup_url(invitation_token: @invitation.token))).html_safe %>`

While the yml would look like:

```
invitation:
  html:
    invitation_link_text: "follow this link to sign up"
    join: "If you'd like to join us, please %{invitation_link}."
```

Doing it this way spares the translation team for having to deal with the HTML directly, which reduces the chances of anything getting broken.